components/03-modules/form/_form.scss

```diff
@@ -10,57 +31,37 @@ $form-legend-line-height: 1.2 !default;
 
         .form__heading {
             display: none;
-            line-height: $button-height;
-            margin-right: 20px;
+            line-height: $form-compact-line-height;
+            margin-right: $form-compact-margin;
         }
 
         .form__content {
             display: flex;
         }
 
         .input {
-            margin-bottom: 0;
-            width: 100%;
             display: flex;
             flex-wrap: nowrap;
             align-items: center;
             justify-content: center;
+            width: 100%;
+            margin-bottom: 0;
 
             &__field {
                 width: 100%;
-                line-height: $button-height;
-                height: $button-height;
+                line-height: $form-compact-line-height;
+                height: $form-compact-height;
                 border-top-right-radius: 0;
                 border-bottom-right-radius: 0;
             }
         }
 
         .button {
             display: block;
-            margin: 0 0 0 (-$button-icon-border-width);
+            margin: $form-button-margin;
             &--native-link {
                 margin: 0;
             }
         }
-
-    }
-
-    &__fieldset,
```

Comment:
Follow the rules - https://docs.snowdog.pro/web/frontend/how-to-write-better-code.html

Reply:
modifiers moved